task_12/src/css/desktop/header.css

```diff
@@ -0,0 +1,41 @@
+header {
+  display: grid;
+  width: 100%;
+  background-image: url(../img/header-bg.jpg);
+  font-size: 15px;
+  align-items: center;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr)); }
+
+.timeBlock {
+  margin-left: 60px;
+  cursor: pointer; }
+  .timeBlock:hover {
+    transition: 500ms linear;
+    color: $hover-text-color; }
+
+.logInBlock {
+  margin-right: 60px;
+  cursor: pointer;
+  justify-self: end; }
+  .logInBlock:hover {
```

Comment:
use '&:hover'

Reply:
I prescribed it this way.
When converted from sss to css - changed
<img width="424" alt="screen shot 2018-02-05 at 15 32 00" src="https://user-images.githubusercontent.com/24464487/35807054-3923566a-0a8a-11e8-801d-dac3fd2b6820.png">

Do I need to write something different?

Reply:
@AleshaOleg, pay more attention to the file extension. It's compiled one.